modules/Bitfinex.py

```diff
@@ -19,9 +19,10 @@ def __init__(self, cfg, log):
         self.cfg = cfg
         self.log = log
         self.lock = threading.RLock()
-        self.req_per_min = 60
-        self.req_period = 15 # seconds
-        self.req_per_period = int(self.req_per_min / ( 60.0 / self.req_period))
+        self.req_per_period = 1
```

Comment:
This disables the ringbuffer, meaning that requests aren't dynamically limited but are sent at a fixed rate.